app/utils/parse/parse-tensorflow.js

```diff
@@ -7,21 +7,23 @@ const returnError = require('../error/return-error')
  * @throws {Error} - Error object
  */
 const parseTensorFlowResult = function(result) {
-    if (result) {
+    if (result && result instanceof String) {
```

Comment:
instanceof will be buggy for primitives. Maybe use the condition 'typeof result === 'string''